database/migrations/2023_09_13_200913_fix_asset_model_min_qty_nullability.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class FixAssetModelMinQtyNullability extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('models', function (Blueprint $table) {
+            $table->integer('min_amt')->nullable()->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('models', function (Blueprint $table) {
+            $table->integer('min_amt')->default(null)->change();
```

Comment:
Making this adjustment will actually make the column non-nullable again.
```suggestion
            $table->integer('min_amt')->nullable(false)->change();
```